src/site/filters/liquid.js

```diff
@@ -233,6 +233,14 @@ module.exports = function registerFilters() {
     return JSON.stringify(id);
   };
 
+  liquid.filters.sortMainFacility = item => {
+    const sorted = item.sort((a, b) => {
+      const sorter = a.entityId - b.entityId;
```

Comment:
This looks fine, but I'm wondering if sorting by the `entityId` is strictly speaking required.

I only ask because this will affect the work we're doing on redoing the CMS -> vets-website integration.

If it _is_ required, we'll just have to write up another ticket to modify the tome-sync export, so that should be fine (:crossed_fingers:). We just need to know.

Reply:
https://github.com/department-of-veterans-affairs/va.gov-team/issues/3599